flink-libraries/flink-table/src/main/scala/org/apache/flink/table/validate/FunctionCatalog.scala

```diff
@@ -190,11 +192,14 @@ object FunctionCatalog {
     // array
     "cardinality" -> classOf[ArrayCardinality],
     "at" -> classOf[ArrayElementAt],
-    "element" -> classOf[ArrayElement]
+    "element" -> classOf[ArrayElement],
 
+    "procTime" -> classOf[CurrentTimestamp]
```

Comment:
make `procTime` lowercase, i.e., `proctime` to keep it consistent with `rowtime`.